tools/crm_mon_print.c

```diff
@@ -2090,11 +2091,12 @@ print_stonith_pending(mon_state_t *state, stonith_history_t *history, unsigned i
                 break;
         }
 
+        history = stonith__sort_history(history);
         for (hp = history; hp; hp = hp->next) {
             if ((hp->state == st_failed) || (hp->state == st_done)) {
                 break;
             }
-            print_stonith_action(state, hp, mon_ops, NULL);
+            print_stonith_action(state, hp, mon_ops, history);
```

Comment:
Note: I wasn't quite clear which of these calls needed a real value and which did not.

Reply:
I don't think it matters; it's only used in print_stonith_action() when event->state is st_failed. So when printing pending and successful actions it will be ignored whether it's passed or not.